src/IB/IBStandardForceGen.cpp

```diff
@@ -759,23 +759,23 @@ IBStandardForceGen::initializeSpringLevelData(std::set<int>& nonlocal_petsc_idx_
     const int num_local_nodes = static_cast<int>(local_nodes.size());
 
     // Determine how many springs are associated with the present MPI process.
-    unsigned int num_springs = 0;
+    unsigned int total_num_springs = 0;
```

Comment:
Also prefix other "total" numbers of things with `total`?

Reply:
Is there something else in this function which could be renamed in this way? I don't see it.

Reply:
Is this change because `num_springs` is used elsewhere in this function? Otherwise, I would prefer to leave it as `num_springs`.

Reply:
For instance, lower down in this file, should it be `total_num_beams` instead of `num_beams`?

I don't care if this is prefixed by `total` or not, but I do care that names are consistent. E.g., `total_num_springs` and `total_num_beams`, or `num_springs` and `num_beams`, but not `total_num_springs` and `num_beams`.

Reply:
Good idea - I'll get those too.

Reply:
There would also be `total_num_target_points`.

Reply:
Got it :)